webpack.config.js

```diff
@@ -1,11 +1,22 @@
 const path = require('path');
 
-module.exports = [
-    buildJsFromTo({from: 'plugins/plugin.archive_analytics.js', to: 'BookReader/plugins/plugin.archive_analytics.js'}),
-    buildJsFromTo({from: 'plugins/menu_toggle/plugin.menu_toggle.js', to: 'BookReader/plugins/plugin.menu_toggle.js'}),
-    buildJsFromTo({from: 'plugins/tts/plugin.tts.js', to: 'BookReader/plugins/plugin.tts.js'}),
+const listOfFiles = [
+    'plugins/plugin.archive_analytics.js',
+    'plugins/menu_toggle/plugin.menu_toggle.js',
+    'plugins/tts/plugin.tts.js',
 ];
 
+const compileFiles = async () => {
```

Comment:
Let's not make this rely on closure to get access to the files it's compiling; makes it a little easier to understand what it's doing when looking at it in isolation :)

Oh, also I don't think think this needs to be `async`; no promises

Also, let's call this `buildJsFiles`, so that it's consistent with the 

```suggestion
/**
 * @param {String[]} files
 */
function buildJsFiles(files) {
```

Reply:
that's fine.  I am ok with changing the name of the top level function but I want to keep the webpack config separate from the files.